src/workers/newNotificationV2Mail.ts

```diff
@@ -856,6 +857,29 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       keyword: keyword?.flags?.title || keyword.value,
     };
   },
+  user_gifted_plus: async (con, user) => {
+    const { subscriptionFlags } = await con.getRepository(User).findOneOrFail({
+      where: {
+        id: user.id,
+      },
+      select: ['subscriptionFlags'],
+    });
+
+    if (!subscriptionFlags?.gifterId) {
```

Comment:
```suggestion
    if (!subscriptionFlags.gifterId) {
```

Reply:
Could not remove this, `subscriptionFlags` may be undefined here but I can remove the prev one ✔️

Reply:
ah right, cool 👍